Backend/ChatAPI.Data/Models/ChatRoomUser.cs

```diff
@@ -0,0 +1,10 @@
+namespace ChatAPI.Data.Models;
+
+public class ChatRoomUser
+{
+    public int ChatRoomId { get; set; }
+    
```

Comment:
Remove empty lines between properties, it makes harder to see all fields without scrolling when we have domain classes with a lot of fields. But you can put empty lines to group related fields.